modules/ocf_printhost/manifests/cups.pp

```diff
@@ -0,0 +1,48 @@
+class ocf_printhost::cups {
+  package { [ 'cups', 'cups-bsd' ]: }
+  ocf::repackage { 'hplip':
+    recommends => false
+  }
+  service { 'cups': }
+
+  File {
+    require => Package['cups'],
+    notify  => Service['cups'],
+  }
+
+  file {
+    '/etc/cups/cupsd.conf':
+      content => template('ocf_printhost/cups/cupsd.conf.erb');
+    '/etc/cups/cups-files.conf':
+      content => template('ocf_printhost/cups/cups-files.conf.erb');
+    '/etc/cups/lpoptions':
+      content => 'Default double';
+    ['/etc/cups/raw.convs', '/etc/cups/raw.types']:
+      content => '# deny printing raw jobs';
```

Comment:
I figured as much. I'm okay with that.

vim actually puts a trailing newline in every single file you edit, so it's unlikely for us to ever commit an actual file like that.